teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -82,7 +85,7 @@ public void configure(TekuConfiguration.Builder builder) {
                 .validatorPerformanceTrackingMode(validatorPerformanceTrackingMode)
                 .validatorExternalSignerSlashingProtectionEnabled(
                     validatorExternalSignerSlashingProtectionEnabled)
-                .graffiti(graffiti));
+                .graffitiProvider(() -> Optional.of(graffiti)));
```

Comment:
This reproduces the current behavior (users can optionally supply a static graffiti value).  If you want to make the new file-based option available, we'll need to add another instance variable to `ValidatorOptions` that accepts the graffiti file path.  For a similar example, see the `--data-path` annotations [here](https://github.com/ConsenSys/teku/blob/2e3a35a4423734cb7de422eecba5dc6eabb996ec/teku/src/main/java/tech/pegasys/teku/cli/options/DataOptions.java#L24-L29).

Then this can become:
```suggestion
                .graffitiProvider(new FileBackedGraffitiProvider(Optional.ofNullable(graffiti), Optional.ofNullable(graffitiFile)));
```